src/algorithm/mixed_radix.rs

```diff
@@ -103,6 +104,14 @@ impl<T: FftNum> MixedRadix<T> {
                 width_outofplace_scratch,
             );
 
+        let height_fft_immut = height_fft.get_immutable_scratch_len();
+        let width_fft_immut = width_fft.get_immutable_scratch_len();
+
+        let immut_scratch_len = 2 * max(
```

Comment:
Same scratch size analysis here as with good-thomas algorithm

Reply:
Although now that I read this more closely, it looks like mixed radix swaps width and height compared to mixed radix, so those will have to be swapped here too.